### DISCUSSION TITLE
Make All notes / Archived / Trash sticky (visible at all times)

### DISCUSSION THREAD

**Discussion by lnolan5017** [standardnotes/forum#2563] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
When I scroll down and select a tag to filter my notes, the All notes / Archived / Trashed scrolls out of view. When I want to return to All notes, I am forced to scroll back up to the top of the Views pane.

**Describe the solution you'd like**
Make All notes / Archived / Trash visible at all times. The Tags title should also remain and then the tags would scroll below that.

**Comment by JaspalSuri** [comment]
Hi @lnolan5017, thanks for the suggestion! I can see how this would be useful.
